docs/tutorial/from-zero-to-hero-write-your-first-kubernetes-charm/expose-operational-tasks-via-actions.md

```diff
@@ -3,19 +3,17 @@
 
 > <small> {ref}`From Zero to Hero: Write your first Kubernetes charm <from-zero-to-hero-write-your-first-kubernetes-charm>` > Expose operational tasks via actions </small>
 >
-> **See previous: {ref}`Preserve your charm's data <preserve-your-charms-data>`** 
-
+> **See previous: {ref}`Integrate your charm with PostgreSQL <integrate-your-charm-with-postgresql>`**
 
 ````{important}
 
 This document is part of a  series, and we recommend you follow it in sequence.  However, you can also jump straight in by checking out the code from the previous branches:
 
-
 ```text
 git clone https://github.com/canonical/juju-sdk-tutorial-k8s.git
 cd juju-sdk-tutorial-k8s
-git checkout 05_preserve_charm_data
-git checkout -b  06_create_actions 
+git checkout 03_integrate_with_psql
```

Comment:
Should we get rid of the "nn_" prefixes on the branch names to avoid having to rename things (more than the once)?

Reply:
I wondered about that too. Adding (especially inserting) or removing a chapter is annoying, because there's this chunk at the start, plus a similar one at the end, as well as previous/next links - and this repo and the tutorial one need to be kept in sync. I think we can probably automate a bunch of that now that we don't have to work with Discourse.

On the other hand, it is nice to have a clear order, and that's missing from the docs now that they don't have leading numbers. We probably aren't adding/removing pages all that often.